applications/API/endpoints.py

```
@@ -0,0 +1,538 @@
+from loguru import logger
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.authentication import TokenAuthentication
+import rest_framework.decorators
+from rest_framework.generics import (
+    GenericAPIView,
+    ListAPIView,
+    RetrieveUpdateDestroyAPIView,
+    CreateAPIView,
+)
+from django.contrib.auth import get_user_model
```

Pylint (3.3.7)

[convention] Imports from package django are not grouped

(C0412)